profiling/src/tracy_impl.rs

```diff
@@ -35,6 +35,20 @@ macro_rules! scope {
     };
 }
 
+#[macro_export]
+macro_rules! function_scope {
+    () => {
+        $crate::tracy_client::span!();
+    };
+    ($data:expr) => {
+        let location = $crate::tracy_client::span_location!();
+        let tracy_span = $crate::tracy_client::Client::running()
+            .expect("function! without a running tracy_client::Client")
```

Comment:
This should be function_scope! without a...

Reply:
"function! without a running tracy_client::Client" should be "function_scope!...